util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -1157,6 +1167,16 @@ module chip_${top["name"]}_${target["name"]} #(
     .RvCoreIbexPipeLine(1),
     .SramCtrlRetAonInstrExec(0),
     .UsbdevRcvrWakeTimeUs(10000),
+  % if lib.num_rom_ctrl(top["module"]) > 1:
+    // TODO(opentitan-integrated/issues/251):
+    // Enable hashing below once the build infrastructure can
+    // load scrambled images on FPGA platforms. The DV can
+    // already partially handle it by initializing the 2nd ROM
+    // with random data via the backdoor loading interface - it
+    // can't load "real" SW images yet since that requires
+    // additional build infrastructure.
+    .SecRomCtrl1DisableScrambling(1),
```

Comment:
Is this still an issue?

Reply:
I think yes. In general, I don't want to make changes on these carry-over PRs.